Data Structures/Graph/Graph.java

```diff
@@ -0,0 +1,196 @@
+import java.util.Scanner;
+import java.util.ArrayList;
```

Comment:
Use of any Collection framework class or interface is not allowed. Please check the problem statement. You can use your own made Queue here instead.